_studio/mfx_lib/decode/vp8/src/mfx_vp8_dec_decode_common.cpp

```diff
@@ -73,7 +73,8 @@ namespace VP8DecodeCommon
         {
             // set offset, but leave last six bytes
             // since they can be start bytes of start code and frame tag
-    
+            if(n_bytes_offset < 6)
```

Comment:
Could we add check at parsing not use this value?

Reply:
Hi yan, thanks for your review.
`n_bytes_offset` is obtained here, msdk don't find the start code when parser this error clip, so we need to set offset, but leave last six bytes, because they can be start bytes of start code and frame tag.